src/pages/New.js

```diff
@@ -22,23 +22,29 @@ const AddCircleIcon = styled(Add)`
 
 export default function New() {
   function handleAccept() {
-    postRestaurant({
+    return postRestaurant({
       imgSrc:
         "https://static.lieferando.de/images/restaurants/de/OPROO0P/logo_465x320.png",
-      title: "Pasta Pronto",
+      title: `${Input}`,
```

Comment:
`Input` is already a string. So there is no need to create a new string.
Try: `title: Title`.